monai/networks/nets/hovernet.py

```diff
@@ -510,12 +492,12 @@ def __init__(
 
     def forward(self, x: torch.Tensor) -> Dict[str, torch.Tensor]:
 
-        if self.mode == 1:
+        if self.mode == HoVerNetMode.ORIGINAL.value:
```

Comment:
it's possible to compare withouth the `.value`:
```py
>>> "ORIGINAL" == monai.utils.enums.HoVerNetMode.ORIGINAL
True
```
would be interesting to see the compatibility of it with torchscript

Reply:
You're right @wyli, and that was the one of the motivations to create StrEnum (instead of Enum) but it seems that TorchScript does not recognize it, and that's one of the reasons for this PR.
I have reported a bug for this on pytorch (https://github.com/pytorch/pytorch/issues/85893).